pkg/sync/adapter.go

```diff
@@ -31,8 +32,8 @@ type Adapter struct {
 	common.AdapterContext
 }
 
-// SyncFilesBuild sync the local files to build container volume
-func (a Adapter) SyncFilesBuild(buildParameters common.BuildParameters, compInfo common.ComponentInfo) (syncFolder string, err error) {
+// SyncsFilesBuild sync the local files to build container volume
```

Comment:
Should be `SyncFilesBuild`, this'll throw a warning because of your additional 's'. I don't mean to be picky, but you did say be brutal!

Reply:
I thought I change that :(, will update

Reply:
I had changed it on my branch that you'd merged, so maybe it changed back during a merge conflict or something?